python/lsst/ip/diffim/utils.py

```diff
@@ -820,14 +820,40 @@ def fitDipoleSource(self, source, **kwds):
         fitResult = alg.fitDipole(source, **kwds)
         return fitResult
 
-    def detectDipoleSources(self, doMerge=True, diffim=None, detectSigma=5.5, grow=3):
+    def detectDipoleSources(self, doMerge=True, diffim=None, detectSigma=5.5, grow=3, weighting=True,
+                            minBinSize=32):
         """!Utility function for detecting dipoles.
 
         Detect pos/neg sources in the diffim, then merge them. A
         bigger "grow" parameter leads to a larger footprint which
         helps with dipole measurement for faint dipoles.
-        """
 
+        Parameters
+        ----------
+        doMerge : `bool`
+           Wether to merge the positive and negagive detections into a single source table
+        diffim : `lsst.afw.image.exposure.exposure.ExposureF`
+           Difference image on which to perform detection
+        detectSigma : `float`
+           Threshold for object detection
+        grow : `int`
+           Number of pixels to grow the footprints before merging
+        weighting : `bool`
+           Wether to use weighting in the background (re)estimation (only applies is useApprox=True)
+        minBinSize : `int`
+           Minimum bin size for the background (re)estimation (only applies if the default leads to
+           nBinX, nBinY < fit order so the default config parameter needs to be decreased, but not
```

Comment:
I think it's clearer to actually use "min" in this docstring, ie "min(nBinX, nBinY) < fit order", or "the smaller of nBinX and nBinY is less than the fit order". (I think that's what you're actually testing, right?)